api/prisma/seed-helpers/translation-factory.ts

```diff
@@ -177,7 +177,7 @@ const translations = (jurisdictionName?: string, language?: LanguagesEnum) => {
           'Lottery results for %{listingName} have been published to applicant accounts.',
       },
       lotteryAvailable: {
```

Comment:
looks like we are updating this copy, right? @ludtkemorgan 
<img width="959" height="147" alt="image" src="https://github.com/user-attachments/assets/57c360b9-d30a-40d9-8658-bd4f7580f522" />